scss/_styles.scss

```diff
@@ -1,6 +1,25 @@
 @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;700&display=swap');
 @import 'colors';
 
+%content {
+  color: $dark-grey;
+  font-size: 16px;
+}
+
+%title {
+  font-size: 16px;
+  font-weight: 700;
+}
+@mixin card-shadow {
+  background-color: $white;
+  box-shadow: 5px 5px 7px 1px $soft-black;
+}
```

Comment:
Move this to a file of mixins, and import the file.

Reply:
Done 😄